localstack/services/cloudformation/models/awslambda.py

```diff
@@ -465,3 +466,76 @@ def _arn(params, resources, resource_id, **kwargs):
                 },
             },
         }
+
+
+# TODO: test
+class LambdaLayerVersion(GenericBaseModel):
```

Comment:
These have been moved here from -ext since the layer / layer permission CRUD operations are now available in community as well.